lib/render/news.php

```diff
@@ -58,7 +58,7 @@ function RenderNewsHeader(News $newsData): void
     echo "<div class='newstext shadowoutline absolute' style='width: 90%; top: 40px; left:10px;'>$payload</div>";
 
     // Author
-    echo "<div class='newsauthor shadowoutline absolute' style='width: 470px; top: 196px; left:0; text-align: right'>~~$authorLink, $niceDate</div>";
+    echo "<div class='newsauthor shadowoutline absolute' style='width: 470px; top: 196px; left:0; text-align: right'>$authorLink, $niceDate</div>";
```

Comment:
Unrelated to this PR, I think we need to limit the text area of the carousel so the user/date don't overwrite the text.  It's even more awkward with the icon now.

![image](https://user-images.githubusercontent.com/32680403/192917735-82746870-15b0-417a-ad96-f5567bf8df54.png)

Reply:
Removed the avatar.

Agreed. Currently it'S in the hands of the news editor to find a suitable length for the post. 

I plan on replacing the carousel with a general news overhaul after our performance hunt is over. Short variants in a new carousel, detail pages with comments, news index, ...